torch/nn/quantized/modules/conv.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+from torch._jit_internal import weak_module, weak_script_method
+from torch._ops import ops
+from torch.nn.modules.module import Module
+from torch.nn.modules.utils import _pair
+from torch.nn.quantized import functional as qF
+
+"""Computes the output shape given convolution parameters."""
+def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
+                       output_padding=0):
+    return np.floor((input_size + 2 * padding - kernel_size - (kernel_size - 1)
+                    * (dilation - 1)) / stride) + 2 * output_padding + 1
+
+@weak_module
+class _ConvNd(Module):
+    def __init__(self, weight, bias, scale, zero_point, dtype,
```

Comment:
Second Jerry's comment. Please change API to match that of linear. The constructor should take the same arguments as float, with extra arguments corresponding to quantization.